x-pack/plugins/uptime/public/components/common/header/action_menu_content.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiButtonEmpty, EuiHeaderLinks, EuiHeaderLink, EuiToolTip } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { useHistory } from 'react-router-dom';
+import { createExploratoryViewUrl, SeriesUrl } from '../../../../../observability/public';
+import { useKibana } from '../../../../../../../src/plugins/kibana_react/public';
+import { useUptimeSettingsContext } from '../../../contexts/uptime_settings_context';
+import { useGetUrlParams } from '../../../hooks';
+import { ToggleAlertFlyoutButton } from '../../overview/alerts/alerts_containers';
+import { SETTINGS_ROUTE } from '../../../../common/constants';
+import { stringifyUrlParams } from '../../../lib/helper/stringify_url_params';
+
+const ADD_DATA_LABEL = i18n.translate('xpack.uptime.addDataButtonLabel', {
+  defaultMessage: 'Add data',
+});
+
+const ANALYZE_DATA = i18n.translate('xpack.uptime.analyzeDataButtonLabel', {
+  defaultMessage: 'Analyze data',
+});
+
+const ANALYZE_MESSAGE = i18n.translate('xpack.uptime.analyzeDataButtonLabel.message', {
+  defaultMessage:
+    'EXPERIMENTAL - Analyze Data allows you to select and filter result data in any dimension and look for the cause or impact of performance problems.',
+});
+
+export function ActionMenuContent(): React.ReactElement {
+  const kibana = useKibana();
+  const { basePath } = useUptimeSettingsContext();
+  const params = useGetUrlParams();
+  const { dateRangeStart, dateRangeEnd } = params;
+  const history = useHistory();
+
+  const syntheticExploratoryViewLink = createExploratoryViewUrl(
+    {
+      'synthetics-series': {
+        dataType: 'synthetics',
+        time: { from: dateRangeStart, to: dateRangeEnd },
+      } as SeriesUrl,
+    },
+    basePath
+  );
+
+  return (
+    <EuiHeaderLinks>
+      <EuiHeaderLink>
+        <ToggleAlertFlyoutButton />
+      </EuiHeaderLink>
+      <EuiHeaderLink>
+        <EuiButtonEmpty
+          aria-label={i18n.translate('xpack.uptime.page_header.settingsLink.label', {
+            defaultMessage: 'Navigate to the Uptime settings page',
+          })}
+          color="primary"
+          data-test-subj="settings-page-link"
+          href={history.createHref({
+            pathname: SETTINGS_ROUTE,
+            search: stringifyUrlParams(params, true),
+          })}
+          iconType="gear"
+        >
+          <FormattedMessage id="xpack.uptime.page_header.settingsLink" defaultMessage="Settings" />
```

Comment:
@justinkambic @formgeist have we decides on the order of buttons? in APM settings comes before Alerts. Here alerts is before Settings, we should use same order for same kind of buttons.

Reply:
![image](https://user-images.githubusercontent.com/3505601/119513049-2b956880-bd74-11eb-8ba5-d305547a6f17.png)

Reply:
@shahzad31 @justinkambic The correct order is described in the latest issue that relates to the menu options https://github.com/elastic/kibana/issues/99902

Reply:
I was following the screenshot in the issue. I prefer to not have settings be the first thing in the list because I tend to not think of Settings as the most important item in nav menus, but I agree we need to have uniformity here.

@formgeist should we follow the ordering of APM for this change, and then create a new issue to switch the buttons across Observability apps? Alternatively we can just keep settings in the front.

Reply:
I just pushed 452d11f2cce59d8af0aab0ae8d70e2dbf6d2ed5c which switches the settings and alerts options.

![image](https://user-images.githubusercontent.com/18429259/119516265-9f1f8180-bd44-11eb-8627-917131b16eff.png)

Reply:
> @formgeist should we follow the ordering of APM for this change, and then create a new issue to switch the buttons across Observability apps? Alternatively we can just keep settings in the front.

We have separate issues that address this for the other Observability apps.

Reply:
@formgeist so you want me to revert the change and have **Alerts** before **Settings**?

Reply:
@justinkambic Sorry, I didn't make that clear. Settings will go in front of alerts like the design suggested.

Reply:
Ok, sounds like it's configured the way we want then.